ParserHelpers.h

```diff
@@ -1,17 +1,13 @@
 #ifndef PARSER_HELPERS_H
 #define PARSER_HELPERS_H
 #include <map>
+#include "CommonRegexes.h"
 #include "Parser.h"
 
 
 namespace commonItems
 {
 
-// const std::string catchallRegex{R"([a-zA-Z0-9_\.\-:@\'\"]+)"}; Leaving this here in case we need to revert.
```

Comment:
If we're leaving registerRegex, we should also leave this.

Reply:
That's like leaving file.txt.old.old

Reply:
Well, the whole block (github's reviews don't let me make that clear). The comment isn't important, but the string below it is.

Reply:
The question is, are we leaving registerRegex?

Reply:
Possibly. I've that this ugly thing that ctre didn't like at first try: https://github.com/ParadoxGameConverters/Vic2ToHoI4/blob/master/Vic2ToHoI4/Source/V2World/Technology/InventionsFactory.cpp#L10

It's possible there's an easy fix, I haven't looked yet.

Reply:
Yeah, it found some "errors" even in the commonItems regexes. It doesn't like escaping backslashes when they aren't needed. Try this: `R"([a-zA-Z0-9_.\xe8\xe9\xf6\xfc:&]+)"`.

Reply:
That builds, but doesn't properly match invention names. Nor does
```
"[a-zA-Z0-9_.\xe8\xe9\xf6\xfc:&]+"
```
or
```
"[a-zA-Z0-9_.\\xe8\\xe9\\xf6\\xfc:&]+"
```

Reply:
However, as @Zemurin pointed out, catchallRegexMatch works just fine.